packages/gatsby-source-filesystem/package.json

```diff
@@ -1,37 +1,41 @@
 {
-  "name": "gatsby-source-filesystem",
-  "description": "Gatsby plugin which parses files within a directory for further parsing by other plugins",
-  "version": "1.5.29",
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
```

Comment:
We use https://www.npmjs.com/package/fixpack for ordering our package.json files.